Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.0-alpine
+LABEL authors="serhiik"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /cinema_service
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
```

Comment:
The `RUN mkdir -p /files/media` command is creating a directory that is later owned by `my_user`. Ensure that this directory is necessary and correctly configured for your application's needs.